src/app/api/quality_matrix/models.py

```diff
@@ -18,3 +22,9 @@ class Timeline(Base):
     id = Column(Integer, primary_key=True, index=True)
     timestamp = Column(Integer, nullable=False)
     quality_matrix = Column(JSON, nullable=False)
+    form = Column(Text, nullable=False)
+
+
+class Forms(str, Enum):
```

Comment:
docstring proposal:

"""
The form of a quality matrix specifies what will be used as grouping and hence defines the vertical (or horizontal?) axis of the table.
E.g. for REPLICATION_SOURCE, every row will correspond to a specific replication source, and the columns will spawn other meta data fields.  The values in the cells will correspond to the counts/percentages where those fields are empty / missing _given_ the replication source of the row.
""""

Reply:
Mostly taken, thanks for suggesting :)